src/components/Lists/ListBase/index.tsx

```diff
@@ -0,0 +1,29 @@
+import {
+    List as MaterialList,
+    ListProps as MaterialListProps,
+    Box as MaterialBox,
+} from "@material-ui/core";
+import { useTheme } from "@material-ui/core/styles";
+
+export interface ListBaseProps extends MaterialListProps {
+    width?: number;
+}
+
+const ListBase = (props: ListBaseProps) => {
+    const theme = useTheme();
+    const { width, ...rest } = props;

```

Comment:
I think width should be 260 by default
```suggestion
    const { width = 260, ...rest } = props;

```